src/common/store/runtime/types.ts

```diff
@@ -18,11 +18,11 @@ export enum AuthProcStages {
 
 export interface IAuthProcState {
   stage: AuthProcStages;
-  provider: AuthProviders;
+  provider?: AuthProviders;
   msg?: string;
 }
 
 export interface IRuntimeState {
-  user: IUserState;
+  // user: IUserState;
```

Comment:
Intended?

Reply:
Actually I meant to remove it altogether, as we do not store any runtime user data, so I'd better just remove it for now.